relay-server/src/actors/outcome.rs

```diff
@@ -424,6 +431,10 @@ impl TrackRawOutcome {
             quantity: Some(msg.quantity),
         }
     }
+
+    fn is_billing(&self) -> bool {
+        matches!(self.outcome, 0 | 2)
```

Comment:
This is hard-coded right now, but we should be using constants instead. I haven't found a nice way to place these constants, ideally we'd have another enum. There is `OutcomeType` already, should we be using this one? It's incomplete, though.

Alternatively, we could have this as associated constants to some type, which would give us the benefit of being able to match on the `u8`. However, then we have the variants of outcomes declared three times: `Outcome`, `OutcomeType`, and the new constants.

Reply:
Can we make `OutcomeType` complete and change the type of `TrackRawOutcome::outcome` to `OutcomeType`, while still serializing it as an integer? https://serde.rs/enum-number.html

Reply:
Yes, can do. What would you expect to happen when an unknown outcome number comes along?

Reply:
I guess drop the outcome? Or do we have a requirement to keep `TrackRawOutcome` agnostic? I imagine that when we introduce a new outcome type, we'd update the upstream relay first, so we would not accidentally drop any outcomes.